spec/02-integration/20-wasm/01-admin-api_spec.lua

```diff
@@ -387,4 +388,162 @@ describe("WASMX admin API [#" .. strategy .. "]", function()
 
 end)
 
+describe("WASMX admin API - wasm = off [#" .. strategy .. "]", function()
+  local admin
+  local bp, db
+  local service
+
+  lazy_setup(function()
+    bp, db = helpers.get_db_utils(strategy, {
+      "routes",
+      "services",
+    })
+
+    service = assert(db.services:insert {
+      name = "wasm-test",
+      url = "http://wasm.test",
+    })
+
+    assert(helpers.start_kong({
+      database = strategy,
+      nginx_conf = "spec/fixtures/custom_nginx.template",
+      wasm = "off",
+    }))
+
+    admin = helpers.admin_client()
+  end)
+
+  describe("/filter-chains", function()
+
+    describe("POST", function()
+      it("returns 400", function()
+        local res = admin:post("/filter-chains", {
+          headers = { ["Content-Type"] = "application/json" },
+          body = {
+            filters = { { name = "tests" } },
+            service = { id = service.id },
+          },
+        })
```

Comment:
nit: you could update this to use the `json()` helper that the other test cases are using:

```suggestion
        local res = admin:post("/filter-chains", json {
          filters = { { name = "tests" } },
          service = { id = service.id },
        })
```